compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.mpp.demo.components.layers
+
+import androidx.compose.mpp.demo.Screen
+
+/**
+ * Test cases for platform layers
+ */
+val PlatformLayers = Screen.Selection(
+    "PlatformLayers",
```

Comment:
It's not about "PlatformLayers", it's common code. I'd suggest just adding a new button to existing `DialogExample` page

Reply:
I want to collect cases to test platformLayers=true/false and compare it with Desktop.

Reply:
Maybe we can move it to another page after stabilizing platformLayers?

Reply:
Decided to place these examples on 2 pages